actions/packages.py

```diff
@@ -153,13 +153,15 @@ def update(self, package_list=[]):
             args.extend(package_list)
         return self.__execute(args)
 
-    def patch(self, updatestack_only=False):
+    def patch(self, updatestack_only=False, allow_vendor_change=False):
         args = ["-n", "-x", "patch", "--auto-agree-with-licenses"]
 
         if self.download_only:
             args.append("--download-only")
         if updatestack_only and self.dup_version == 2:
             args.append("--updatestack-only")
+        if allow_vendor_change:
+            args.append("--allow-vendor-chnage")
```

Comment:
A typo here and also a question:

Shouldn't we check if the `--allow-vendor-change` is compatible for the Zypper version that is running? 

Something like:

```suggestion
        if allow_vendor_change and self.dup_version == 2:
            args.append("--allow-vendor-change")
```

That is actually what we currently do, for instance for `packages.dup`:

https://github.com/openSUSE/zypp-plugin-spacewalk/blob/dab6ccfdb54caa03eb1ab25d894a4efdbd29c073/actions/packages.py#L178-L182

Reply:
dup version 2 is something different. It has nothing to do with zypper able to know this option or not.